de-lib/src/main/java/org/iplantc/de/analysis/client/views/dialogs/AnalysisUserSupportDialog.java

```diff
@@ -0,0 +1,237 @@
+import com.google.gwt.event.logical.shared.ValueChangeHandler;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.gwt.user.client.ui.Label;
+import com.google.gwt.user.client.ui.TextArea;
+import com.google.inject.Inject;
+
+import com.sencha.gxt.core.client.dom.ScrollSupport;
+import com.sencha.gxt.core.client.util.Margins;
+import com.sencha.gxt.core.client.util.ToggleGroup;
+import com.sencha.gxt.widget.core.client.Window;
+import com.sencha.gxt.widget.core.client.button.TextButton;
+import com.sencha.gxt.widget.core.client.container.VerticalLayoutContainer;
+import com.sencha.gxt.widget.core.client.event.SelectEvent;
+import com.sencha.gxt.widget.core.client.form.CheckBox;
+import com.sencha.gxt.widget.core.client.form.FieldLabel;
+import com.sencha.gxt.widget.core.client.form.FormPanel;
+import com.sencha.gxt.widget.core.client.form.Radio;
+
+import java.util.Date;
+
+
+/**
+ * @author sriram
+ */
+public class AnalysisUserSupportDialog extends Window {
+
+    private TextArea comments;
+    private TextButton submitBtn;
+
+    public interface AnalysisUserSupportAppearance {
+
+        String outputUnexpected();
+
+        String noOutput();
+
+        String backgroudColor();
+
+        String selectCondition();
+
+        String comments();
+
+        String needHelp();
+
+        String submit();
+
+        String agreeToShare();
+
+        String disclaimer();
+
+        String termsOfSupport();
+
+        String supportRequestFailed();
+
+        String supportRequestSuccess();
+
+        String deSystemId();
+    }
+
+
+    private Analysis selectedAnalysis;
+    private HelpRendererTemplates renderer = GWT.create(HelpRendererTemplates.class);
+    private AnalysisUserSupportAppearance appearance = GWT.create(AnalysisUserSupportAppearance.class);
+    private VerticalLayoutContainer vlc;
+    private TextButton needHelpButton;
+    @Inject AnalysesAutoBeanFactory factory;
+
+    @Inject
+    public AnalysisUserSupportDialog() {
+        needHelpButton = new TextButton(appearance.needHelp());
+        needHelpButton.addSelectHandler(new SelectEvent.SelectHandler() {
+            @Override
+            public void onSelect(SelectEvent event) {
+                displayConfirmation();
+            }
+        });
+        submitBtn = new TextButton(appearance.submit());
+        submitBtn.disable();
+        vlc = new VerticalLayoutContainer();
+        vlc.getElement().getStyle().setBackgroundColor(appearance.backgroudColor());
+        vlc.setScrollMode(ScrollSupport.ScrollMode.AUTO);
+    }
+
+    public void renderHelp(Analysis analysis) {
+        this.selectedAnalysis = analysis;
+        HTML text = null;
+        switch (AnalysisExecutionStatus.fromTypeString(selectedAnalysis.getStatus().toLowerCase())) {
+            case SUBMITTED:
+                if (selectedAnalysis.getSystemId().equalsIgnoreCase(appearance.deSystemId())) {
+                    text = new HTML(renderer.renderCondorSubmitted(selectedAnalysis));
```

Comment:
I mean instead of having `HelpRendererTemplates` referenced in this class since it affects the appearance and uses a template, have it referenced in `AnalysisUserSupportDefaultAppearance`.  Then you would add those methods to `AnalysisUserSupportAppearance`.
This line would then become:
text = new HTML(appearance.renderCondorSubmitted(selectedAnalysis));
or even
text = appearance.renderCondorSubmitted(selectedAnalysis);
This way `AnalysisUserSupportDefaultAppearance` has complete control over all of the text and also which templates are used.